phpBB/styles/prosilver/template/quickreply_editor.html

```diff
@@ -8,6 +8,7 @@ <h2 class="quickreply-title">{L_QUICKREPLY}</h2>
 						<dt><label for="subject">{L_SUBJECT}{L_COLON}</label></dt>
 						<dd><input type="text" name="subject" id="subject" size="45" maxlength="124" tabindex="2" value="{SUBJECT}" class="inputbox autowidth" /></dd>
 					</dl>
+				<!-- EVENT quickreply_editor_subject_after -->
 				<!-- EVENT quickreply_editor_message_before -->
```

Comment:
I am not against it, I just included this event for convention, that every _before event has it's _after event, too.